packages/command-palette/src/index.tsx

```diff
@@ -91,12 +98,44 @@ export function CommandMenuGroup() {
 		setEmptyListNotice,
 		navigate,
 		currentRoute,
+		currentSiteId,
+		useSites,
 	} = useCommandPaletteContext();
-	const { commands, filterNotice, emptyListNotice } = useCommandPalette();
+	const { commands, filterNotice, emptyListNotice, singleSiteMode } = useCommandPalette();
+	const { __ } = useI18n();
+
+	const sites = useSites();
+	const currentSite = sites.find( ( site: { ID: unknown } ) => site.ID === currentSiteId );
+	const adminUrl =
+		currentSite?.options?.wpcom_admin_interface === 'wp-admin'
```

Comment:
We have this utility function that handles the admin interface `siteUsesWpAdminInterface( site )`.